keps/sig-testing/2464-kubetest2-ci-migration/kep.yaml

```
@@ -0,0 +1,41 @@
+title: Kubetest2 CI Migration
+kep-number: 2441
+authors:
+- amwat
+owning-sig: sig-testing
+participating-sigs:
+- sig-release
```

/cc @wojtek-t 
we'd like your input to move this to implementable
```suggestion
- sig-release
- sig-scalability
```